client/src/main/scala/drt/client/components/FlightComponents.scala

```diff
@@ -85,7 +85,7 @@ object FlightComponents {
           <.div(^.className := "bars",
             splits.map {
               case (paxTypeAndQueue, paxCount) =>
-                val percentage: Double = paxCount.toDouble / splitTotal * 100
+                val percentage: Double = paxCount.toDouble / splitTotal * 80
```

Comment:
why is this 80? Might be worth storing it in a descriptive variable.

Reply:
yeah that crossed my mind when i changed the value. i've called it `divHeightScalingFactor` ...